javascripts/endGame.js

```diff
@@ -0,0 +1,59 @@
+function checkWin() {
+  var winningTiles = $("[data-val=" + winningTileValue + "]");
+  if (winningTiles.length > 0) {
+    priorWin = true;
+    winnerTile(winningTiles);
+    endGame('win');
+  } else { return false;}
```

Comment:
With else statements, you should break those out into their own lines, even if it's a one-liner.